libr/anal/p/anal_x86_cs.c

```diff
@@ -3365,9 +3365,7 @@ static int analop(RAnal *a, RAnalOp *op, ut64 addr, const ut8 *buf, int len, RAn
 #endif
 	//XXX: capstone lcall seg:off workaround, remove when capstone will be fixed
         if (n>=1 && mode == CS_MODE_16 && !strncmp (insn->mnemonic, "lcall", 5)) {
```

Comment:
Spaces around operators

```suggestion
        if (n >= 1 && mode == CS_MODE_16 && !strncmp (insn->mnemonic, "lcall", 5)) {
```